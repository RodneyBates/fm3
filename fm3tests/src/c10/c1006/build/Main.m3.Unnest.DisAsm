 ItkEOF()
 ItkRightEnd()
 ItkModuleLtPatch(C4,L1(16_1),[1,0])
 ItkModuleBeginPatch(C292,L1(16_1),[16,2])
 ItkModuleRt(L1(16_1),[1,0])
 ItkDeclScopeRt(L1(16_1))
 ItkLookupScopeRt(L1(16_1))
 ItkStmtListLt(L0(16_0),[17,2])
 ItkStmtListRt(L0(16_0),[17,2])
 ItkBlockDeclListLtPatch(C10,L4(16_4),[5,2])
 ItkBlockDeclListRt(L4(16_4),[5,2])
 ItkBlockDeclListSepPatch(C116,L3(16_3),[10,0])
 ItkPartialRevealLtPatch(C116,[10,9])
 ItkPartialRevealSubtypePatch(C121,[11,2])
 ItkPartialRevealRt([10,9])
 ItkREFDefLtPatch(C121,[11,5])
 ItkREFDefRt([11,5])
 ItkRecDefLtPatch(C121,L2(16_2),[11,9])
 ItkRecDefRt(L2(16_2),[11,9])
 ItkDeclScopeRt(L2(16_2))
 ItkFieldDeclListLtPatch(C124,L2(16_2),[12,6])
 ItkFieldDeclListRt(L2(16_2),[12,6])
 ItkFieldDeclListSepPatch(C169,L1(16_1),[13,4])
 ItkFieldDeclLtPatch(C169,[13,6])
 ItkFieldDeclTypePatch(C187,[13,9])
 ItkFieldDeclValPatch(C192,[14,4])
 ItkFieldDeclRt([13,6])
 ItkIdRefAtom(Id2"P",[13,11])
 ItkFieldDeclIdListLtPatch(C169,L1(16_1),[13,6])
 ItkFieldDeclIdListRt(L1(16_1),[13,6])
 ItkDeclId(L14(16_e),Id8"F2",[13,6])
 ItkFieldDeclLtPatch(C124,[12,6])
 ItkFieldDeclTypePatch(C142,[12,9])
 ItkFieldDeclValPatch(C147,[13,4])
 ItkFieldDeclRt([12,6])
 ItkIdRefAtom(Id3"R",[12,11])
 ItkFieldDeclIdListLtPatch(C124,L1(16_1),[12,6])
 ItkFieldDeclIdListRt(L1(16_1),[12,6])
 ItkDeclId(L14(16_e),Id7"F1",[12,6])
 ItkDeclScopeLt(L2(16_2))
 ItkIdRefAtom(Id6"R1",[10,9])
 ItkBlockDeclListSepPatch(C70,L2(16_2),[8,0])
 ItkFullRevealLtPatch(C70,[8,9])
 ItkFullRevealEqualsPatch(C78,[8,16])
 ItkFullRevealRt([8,9])
 ItkREFDefLtPatch(C78,[8,18])
 ItkREFDefRt([8,18])
 ItkIdRefAtom(Id3"R",[8,22])
 ItkQualIdAtoms(Id2"P",Id5"Q",[8,9],[8,14])
 ItkBlockDeclListSepPatch(C38,L1(16_1),[6,0])
 ItkSubtypeDeclLtPatch(C38,[6,7])
 ItkSubtypeDeclSubtypePatch(C44,[6,9])
 ItkSubtypeDeclRt([6,7])
 ItkReservedId(Rid"<ROOT>",[6,12])
 ItkDeclId(L8(16_8),Id3"R",[6,7])
 ItkSubtypeDeclLtPatch(C10,[5,7])
 ItkSubtypeDeclSubtypePatch(C16,[5,9])
 ItkSubtypeDeclRt([5,7])
 ItkQualIdAtoms(Id3"R",Id4"Y",[5,12],[5,16])
 ItkDeclId(L8(16_8),Id2"P",[5,7])
 ItkLookupScopeLt(L1(16_1))
 ItkDeclScopeLt(L1(16_1))
 ItkLeftEnd()
 ItkBOF()
     0
