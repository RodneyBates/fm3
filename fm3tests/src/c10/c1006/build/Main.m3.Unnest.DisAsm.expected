 ItkRightEndIncomplete()
 ItkStmtListLt(L0(16_0),[17,2])
 ItkStmtListRt(L0(16_0),[17,2])
 ItkBlockDeclListLtPatch(C9,L4(16_4),[5,2])
 ItkBlockDeclListRt(L4(16_4),[5,2])
 ItkBlockDeclListSepPatch(C115,L3(16_3),[10,0])
 ItkPartialRevealLtPatch(C115,[10,9])
 ItkPartialRevealSubtypePatch(C120,[11,2])
 ItkPartialRevealRt([10,9])
 ItkREFDefLtPatch(C120,[11,5])
 ItkREFDefRt([11,5])
 ItkRecDefLtPatch(C120,L2(16_2),[11,9])
 ItkRecDefRt(L2(16_2),[11,9])
 ItkFieldDeclListLtPatch(C120,L2(16_2),[12,6])
 ItkFieldDeclListRt(L2(16_2),[12,6])
 ItkFieldDeclListSepPatch(C165,L1(16_1),[13,4])
 ItkFieldDeclLtPatch(C165,[13,6])
 ItkFieldDeclTypePatch(C183,[13,9])
 ItkFieldDeclValPatch(C188,[14,4])
 ItkFieldDeclRt([13,6])
 ItkIdRefAtom(Id2"P",[13,11])
 ItkFieldDeclIdListLtPatch(C165,L1(16_1),[13,6])
 ItkFieldDeclIdListRt(L1(16_1),[13,6])
 ItkDeclId(L14(16_e),Id8"F2",[13,6])
 ItkFieldDeclLtPatch(C120,[12,6])
 ItkFieldDeclTypePatch(C138,[12,9])
 ItkFieldDeclValPatch(C143,[13,4])
 ItkFieldDeclRt([12,6])
 ItkIdRefAtom(Id3"R",[12,11])
 ItkFieldDeclIdListLtPatch(C120,L1(16_1),[12,6])
 ItkFieldDeclIdListRt(L1(16_1),[12,6])
 ItkDeclId(L14(16_e),Id7"F1",[12,6])
 ItkIdRefAtom(Id6"R1",[10,9])
 ItkBlockDeclListSepPatch(C69,L2(16_2),[8,0])
 ItkFullRevealLtPatch(C69,[8,9])
 ItkFullRevealEqualsPatch(C77,[8,16])
 ItkFullRevealRt([8,9])
 ItkREFDefLtPatch(C77,[8,18])
 ItkREFDefRt([8,18])
 ItkIdRefAtom(Id3"R",[8,22])
 ItkQualIdAtoms(Id2"P",Id5"Q",[8,9],[8,14])
 ItkBlockDeclListSepPatch(C37,L1(16_1),[6,0])
 ItkSubtypeDeclLtPatch(C37,[6,7])
 ItkSubtypeDeclSubtypePatch(C43,[6,9])
 ItkSubtypeDeclRt([6,7])
 ItkReservedId(Rid"<ROOT>",[6,12])
 ItkDeclId(L8(16_8),Id3"R",[6,7])
 ItkSubtypeDeclLtPatch(C9,[5,7])
 ItkSubtypeDeclSubtypePatch(C15,[5,9])
 ItkSubtypeDeclRt([5,7])
 ItkQualIdAtoms(Id3"R",Id4"Y",[5,12],[5,16])
 ItkDeclId(L8(16_8),Id2"P",[5,7])
 ItkBlockLt(L1(16_1),[5,2])
 ItkLeftEnd()
 ItkBOF()
     0
